pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -458,6 +475,27 @@ func (dswp *desiredStateOfWorldPopulator) markPodProcessed(
 	dswp.pods.processedPods[podName] = true
 }
 
+func (dswp *desiredStateOfWorldPopulator) recordFailedProcessPodEvent(
+	pod *v1.Pod, msg string, logMsg string) {
+	dswp.failedProcessPodMessages.Lock()
+	defer dswp.failedProcessPodMessages.Unlock()
+
+	uniquePodName := util.GetUniquePodName(pod)
+
+	msgMap, found := dswp.failedProcessPodMessages.messages[uniquePodName]
+	if !found {
+		dswp.failedProcessPodMessages.messages[uniquePodName] = make(map[string]time.Time)
+		msgMap = dswp.failedProcessPodMessages.messages[uniquePodName]
+	}
+
+	lastTime, found := msgMap[msg]
+	if !found || time.Since(lastTime) > dswp.failureEventInterval {
+		msgMap[msg] = time.Now()
+		dswp.recorder.Eventf(pod, v1.EventTypeWarning, kevents.FailedProcessVolume, msg)
+		glog.Errorf(logMsg)
```

Comment:
is it a good idea to use verbosity level here:
`glog.V(xx).Errorf(logMsg)` ?

my assumption is that this can happen often in certain conditions, thus the possibility of logging a lot here?